modules/pipeline/conf/conf.go

```diff
@@ -24,9 +24,9 @@ import (
 
 // Conf 定义配置对象.
 type Conf struct {
-	ListenAddr  string `env:"LISTEN_ADDR" default:":3081"`
-	Debug       bool   `env:"DEBUG" default:"false"`
-	DiceCluster string `env:"DICE_CLUSTER" default:"local"` // 服务所在集群
```

Comment:
what's different between `DICE_CLUSTER` and `DICE_CLUSTER_NAME`?

Reply:
not found `DICE_CLUSTER` in envs